e2e/features/case-progression.feature

```diff
@@ -402,6 +402,30 @@ Feature: Case progression
 
     And I should see the option `Upload additional evidence` for the `Next step` field
 
+    # AO:
+
+    When I switch to be a `Admin Officer`
+    And I click the `Overview` tab
+
+    Then I should only see the `caseOfficer_listing` case progress image
```

Comment:
I think it is Admin officer listing..typo

Reply:
'Admin Officer' and 'Case Officer' both share the same case progress image, called 'caseOfficer_listing' i.e. they view the same image. After discussing with Jack, this will most likely remain as they form part of HMCTS, which is different to a third-party legal rep. At this stage no need to duplicate the image.

Reply:
👍